lsp/ts_ls.lua

```diff
@@ -59,13 +59,15 @@ return {
     -- We select then from the project root, which is identified by the presence of a package
     -- manager lock file.
     local root_markers = { 'package-lock.json', 'yarn.lock', 'pnpm-lock.yaml', 'bun.lockb', 'bun.lock' }
-    -- Give the root markers equal priority by wrapping them in a table
-    root_markers = vim.fn.has('nvim-0.11.3') == 1 and { root_markers, { '.git' } }
-      or vim.list_extend(root_markers, { '.git' })
+    if vim.fn.has('nvim-0.11') == 1 then
+      root_markers = { root_markers, { '.git' } }
+    else
+      vim.list_extend(root_markers, { '.git' })
+    end
```

Comment:
why was this changed? do not change this.

- 0.11.3 or older needs `vim.list_extend(root_markers, { '.git' })` which yields: `{ 'package-lock.json', ..., 'git' }`
- 0.11.4+ and newer supports nested lists to signal "equal priority".